language/move-prover/bytecode/src/well_formed_instrumentation.rs

```diff
@@ -35,7 +35,7 @@ impl WellFormedInstrumentationProcessor {
         Box::new(Self {})
     }
 }
-
+#[allow(clippy::redundant-clone)]
```

Comment:
It looks like there's a typo error, and it seems that `#[allow(clippy::redundant_clone)]` is correct.